salt/orchestrate/edx/backup.sls

```diff
@@ -109,28 +94,17 @@ execute_enabled_backup_scripts:
         - salt: deploy_backup_instance_to_{{ ENVIRONMENT }}
         - salt: mount_and_format_backup_drive
 
-unmount_and_detach_backup_drive:
-  salt.state:
-    - tgt: 'G@roles:backups and G@environment:{{ ENVIRONMENT }}'
-    - tgt_type: compound
-    - sls:
-        - backups.unmount_drive
-    - require:
-        - salt: execute_enabled_backup_scripts
-
-terminate_backup_instance_in_{{ ENVIRONMENT }}:
+stop_backup_instance_in_{{ ENVIRONMENT }}:
   salt.function:
-    - name: saltutil.runner
+    - name: cloud.action
     - tgt: 'roles:master'
     - tgt_type: grain
     - arg:
-        - cloud.destroy
+        - stop
```

Comment:
Since we are stopping the instance we should update the profile so that the stop action is to stop and not terminate. The ec2 cloud provider docs in salt should have examples.